packages/embark/src/lib/core/plugins.js

```diff
@@ -164,6 +164,7 @@ Plugins.prototype.getPluginsProperty = function(pluginType, property, sub_proper
 Plugins.prototype.runActionsForEvent = function(eventName, args, cb) {
   if (typeof (args) === 'function') {
     cb = args;
+    args = null;
```

Comment:
These two changes are here because we were not passing error information back in the callback properly when there were no arguments passed back (and `args` was a `function`).